python/test/unit/runtime/test_launch.py

```diff
@@ -204,3 +205,64 @@ def kernel(x):
 
     triton.knobs.runtime.jit_post_compile_hook = None
     triton.knobs.runtime.jit_cache_hook = None
+
+
+def test_pre_run_hooks_jit(device):
```

Comment:
I think we can take out the original `test_pre_run_hooks_jit`. There's no assert here and the new test serves the same purpose